### ISSUE TITLE
New Asset Import Issue

### ISSUE THREAD

**Issue by FunlabStudio** [flutter/flutter#90133]
Hello,
Whenever I import a new asset why do I need to close the application and then run it? Is there any way to import assets while using the application and prevent these issues?

**Comment by maheshmnj** [comment]
Hi @FunlabStudio  By importing Assets do you mean adding an external asset e.g image?

if yes it works with a hot restart. You don't need to close the application.

**Comment by FunlabStudio** [comment]
Yes, I'm trying to load external assets and did restart the application but floating an error.  @maheshmnj 

Unable to load asset: assets/newAssets/changePassword.png. For your Reference

**Comment by maheshmnj** [comment]
Please share the output of `flutter doctor -v` and  a [minimal and complete reproducible code](https://stackoverflow.com/help/minimal-reproducible-example) to investigate the issue further.

Thanks
